Changelog.md

```diff
@@ -4,6 +4,7 @@ Breaking changes:
  * Inline Assembly: Disallow ``.`` in user-defined function and variable names.
  * Inline Assembly: Slot and offset of storage pointer variable ``x`` are accessed via ``x.slot`` and ``x.offset`` instead of ``x_slot`` and ``x_offset``.
  * JSON AST: Remove members with ``null`` value from JSON output.
+ * JSON AST: Mark hex string literals with ``token: "hexString"``.
```

Comment:
Should call it `kind` instead. Should we leave the legacy one unchanged?

Reply:
Yes

Reply:
Yes to what? :)

Reply:
Leave the key unchanged.

Reply:
So what happens to `unicodeStringLiteral` then in #9412? It won't be introduced in the legacy mode?

Reply:
Can't we just keep whatever key is used and modify the value? Legacy does not mean we cannot introduce anything new.

Reply:
I'm not sure what our position is. I think what makes sense is that with the "legacy AST" what we agree not to change is the broader structure, but it would not make any sense to disallow us to change any values in a breaking releases.


At last in the case of the unicode token it would become really confusing to deal with -- I suggest we take the PR in its current form, but maybe getting some feedback from anyone (if anyone?!) using the legacy AST.

Reply:
Of course, the legacy part of the legacy ast is just its structure. Everything else changes in line with the non-legacy AST. That's why I also think there is no big harm in keeping it around.

Reply:
I'm okay with the PR as is, merge it if you agree.